script/configs/M1_deploy_mainnet.config.json

```diff
@@ -1,10 +1,10 @@
 {
      "multisig_addresses": {
-          "communityMultisig": "",
+          "communityMultisig": "0xbeB9364D9B8D155842FB82C1323B3EBEC1552D9D",
           "operationsMultisig": "0xBE1685C81aA44FF9FB319dD389addd9374383e90",
           "pauserMultisig": "0x5050389572f2d220ad927CcbeA0D406831012390",
           "executorMultisig": "0xBE1685C81aA44FF9FB319dD389addd9374383e90",
```

Comment:
executor should be [0x369e6F597e22EaB55fFb173C6d9cD234BD699111](https://etherscan.io/address/0x369e6F597e22EaB55fFb173C6d9cD234BD699111)

Reply:
fixed in https://github.com/Layr-Labs/eigenlayer-contracts/pull/71/commits/3f6b7b04da251f5c53d2af60c2a91da256659e94